src/core/ai_circuit_breaker.py

```
@@ -0,0 +1,53 @@
+        if not state.is_open:
+            return True
+            
+        if datetime.now() - state.opened_at >= self.reset_timeout:
+            self._reset_circuit(model)
+            return True
+            
+        return False
+        
+    def _reset_circuit(self, model: str) -> None:
+        """Reset circuit state for a model."""
+        self.circuits[model] = CircuitState()
```

Ruff (0.8.2)

No newline at end of file

Add trailing newline

(W292)